.ci/jenkins/scripts/minikube.groovy

```diff
@@ -100,8 +100,9 @@ void preChecks() {
         ${minikubeContainerEngine} info
 
         if [[ ! \$(command -v minikube) ]]; then
-            curl -LO https://github.com/kubernetes/minikube/releases/download/v${minikubeVersion}/minikube-${minikubeVersion}-0.x86_64.rpm
-            sudo yum localinstall -y minikube-${minikubeVersion}-0.x86_64.rpm
+            curl -LO https://github.com/kubernetes/minikube/releases/download/v${minikubeVersion}/minikube-linux-amd64
+            sudo cp minikube-linux-amd64 /usr/local/bin/minikube
+            sudo chmod 755 /usr/local/bin/minikube
```

Comment:
We have a script to do this, maybe it's worth reusing it? https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/main/hack/ci/install-minikube.sh

There we are able to fetch the correct minikube version based on the platform, so we avoid hardcoding here. wdyt?

Reply:
I've applied the suggested change. Thanks!